### PR TITLE
sslnew always exits with EXIT failure

### PR DESCRIPTION
**Pull request by Sashan**
Fixes: 5

### CODE REVIEW

File: source/sslnew.c [hunk 1]
```diff
@@ -123,6 +129,7 @@ int main(int argc, char *argv[])
     else
         printf("Average time per SSL/BIO creation call: %lfus\n",
                avcalltime);
+    rc = 0;
```
Comment (quarckster) [inline source/sslnew.c:132]: do we really everything else except this line on this PR?

File: source/sslnew.c [hunk 2]
```diff
@@ -129,6 +129,7 @@ int main(int argc, char *argv[])
...
```
Comment (mattcaswell) [inline source/sslnew.c:132]: Better would be to use `rc = EXIT_SUCCESS` (since we initialise it with `EXIT_FAILURE`)

**Review by mattcaswell — CHANGES_REQUESTED** [review]
It seems we have a spurious additional commit in this PR.

**Review by mattcaswell — APPROVED** [review]

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
Merged. Thank you.

**Pull request closed without merging**